tests/cypress/e2e/hedy_page/back_to_level_button.cy.js

```diff
@@ -2,15 +2,7 @@ import {goToHedyLevel2Page, goToHedyPage} from "../tools/navigation/nav";
 
 describe('Is able to use "Go back to level x" button', () => {
     it('Passes', () => {
-      // Test when code is unchanged
-      goToHedyLevel2Page();
-      cy.get('#prev_level_button').click();
-      cy.url().should('include', Cypress.env('hedy_page'));
-
-      // Test when code is changed
-      goToHedyLevel2Page();
-      cy.get('#editor').type('hello');
-      cy.get('#prev_level_button').click();
-      cy.url().should('include', Cypress.env('hedy_page'));
+      // make sure to text navigating to next levels

```

Comment:
Can you add this a TODO comment with an explanation as to why we are removing this test? Some IDEs gather TODO comments and also add some context for someone who is just reading the code and doesn't have knowledge of this PR.

Reply:
Great will do!